pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json.stdout.txt

```diff
@@ -9,5 +9,7 @@
 <{%reset%}><{%fg 9%}>        [urn=urn:pulumi:dev::test::pulumi-python:dynamic:Resource::test-r]
 <{%reset%}><{%reset%}><{%fg 13%}><{%bold%}>Resources:<{%reset%}>
     1 unchanged
+    <{%fg 1%}>1 errored<{%reset%}>
+
```

Comment:
There's a newline too many here if there are errored resources.  It should have the same amount of newlines (1) no matter if there is an errored resource or if there is not.